CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ For semantic convention package changes, see the [semconv CHANGELOG](packages/se
 
 ### :house: (Internal)
 
+* refactor(instrumentation-fetch): refactor `fetch()` tests for clarity, type safety and realism [#5268](https://github.com/open-telemetry/opentelemetry-js/pull/5268)
```

Comment:
do we do changelog for test only changes? I though it was just for changes affecting final user

Reply:
Happy to remove it! (Though I think the "Internal" category is for those changes that does _not_ affect final users, but in any case agreed that this is probably small enough of a change to be left out)